lib/beaker-pe/install/pe_utils.rb

```diff
@@ -101,6 +101,28 @@ def install_ca_cert_on(host, opts)
           end
         end
 
+        # Return agent nodes with 'lb_connect' role
+        def loadbalancer_connecting_agents
+          lb_connect_nodes = select_hosts(roles: ['lb_connect'])
+          lb_connect_agents = lb_connect_nodes.reject { |h| h['roles'].include?('loadbalancer')}
+        end
+
+        # Returns true if the host is an agents that is configured  to connect to loadbalancer
+        #
+        # @param [Host] host to be validated
+        def is_lb_connect_agent(host)
+          host['roles'].include?('lb_connect')  && ! host['roles'].include?('loadbalancer')
+        end
+
+        # Returns true if loadbalncer exists and is configured with 'lb_connect' role
+        def is_lb_connect_loadbalancer
```

Comment:
the way this method is named points to it needing to have a host passed into it, like `is_lb_connect_agent` above. A better name for this might be `lb_connect_loadbalancer_exists?`. Note that this method should end in a `?` character as well.